docs/user_guide/mitigation.rst

```diff
@@ -68,6 +68,218 @@ Postprocessing
 
 .. currentmodule:: fairlearn.postprocessing
 
+Fairlearn currently supports one postprocessing technique that is based on the
+paper *Equality of Opportunity in Supervised Learning* [#3]_. Unlike other
+mitigation techniques :class:`ThresholdOptimizer` is built to satisfy the
+specified fairness criteria exactly and with no remaining disparity.
+In many cases this comes at the expense of performance, for example, with
+significantly lower accuracy. Regardless, it is a useful data point to compare
+results with.
```

Comment:
Do you want to mention that it also works with a pretrained estimator here? It's sort of implicit in the name, but might be worth mentioning.

Reply:
added that as a new section